cfme/tests/containers/test_provider_crud.py

```diff
@@ -0,0 +1,45 @@
+import uuid
+import pytest
+from utils import testgen
+from utils.providers import setup_a_provider as _setup_a_provider
+from utils.update import update
+from utils.version import current_version
+
+pytestmark = pytest.mark.uncollectif(lambda: current_version() < "5.6")
+
+pytest_generate_tests = testgen.generate(
+    testgen.container_providers, scope="function")
+
+# CMP -9844 # CMP - 9846
+
+
+@pytest.yield_fixture(scope="function")
+def a_container_provider():
+    prov = _setup_a_provider("container")
```

Comment:
IIRC _setup_a_provider does provider.create(). Then there is the fixture "has_no_container_providers" then there is, on ine 34 provider.create()

I think we are doing one unnecessary create here. I think I have solved it in my patch where I do `create=False`
https://github.com/ManageIQ/integration_tests/pull/3308/commits/315005af5b30a86af557e454eab7ed9bfa973b03#diff-87d9a1e8542ceaacf701424df5e7cda8R13
Our tests are doing similar things. We should leave only one of them.